pkg/controller/template/test_data/templates/worker/01-worker-kubelet/libvirt/units/kubelet.service

```diff
@@ -18,7 +18,7 @@ contents: |
         --container-runtime=remote \
         --container-runtime-endpoint=/var/run/crio/crio.sock \
         --allow-privileged \
-        --node-labels=node-role.kubernetes.io/worker \
+        --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_version=${OS_VERSION} \
```

Comment:
same q: why is the worker label changed to master in the `01-worker-kubelet`?

Reply:
ditto ^